scripts/supervisor.py

```diff
@@ -37,7 +35,9 @@ def parse_args(argv):
 
 
 def main():
-    sys.exit(teuthology.dispatcher.supervisor.main(parse_args(sys.argv[1:])))
+    args = parse_args(sys.argv[1:])
+    import teuthology.dispatcher.supervisor
```

Comment:
this is also strange, the supervisor script has only purpose to use teuthology.dispatcher.supervisor.